### DISCUSSION TITLE
Searches and smartlists consider the owner of the list instead of the current User?

### DISCUSSION THREAD

**Discussion by SimonHova** [ampache/ampache#3452] [General]
I saw that in my [bug report](https://github.com/ampache/ampache/issues/3427) you had modified one of the search features in the latest dev, and have made it official with an entry in the [wiki](https://github.com/ampache/ampache/wiki/ampache6-details#searches-and-smartlists-consider-the-owner-of-the-list-instead-of-the-current-user). While it's always great to see a new feature being added to Ampache, I don't know if I'm fully behind this particular change.

In my particular use case, I am one of four users on Ampache, so it's me, my wife, and my two little kids. I have a playlist called **My Favorites** which would play the favorite songs of the current user, so my wife and I can share the same playlist, and it'll play the songs for the logged in user. When the kids are in the car with us, I made another list called **Our Favorites** so it can combine the favorite songs of the logged in user with the songs the kids like. Because they are still young enough where they still enjoy listening to the same songs over and over and over and over again, so they share a list called, **Kids Favorites,** where I use their user names to explicitly call out their favorites, I can do all this with only three playlists.

On this new system, I'm not gaining any new functionality, because I can still link to the favorites of the playlists of a particular user, but I would have to create three additional smart playlists to play the favorites for my wife and each of my two kids, plus another one that combines the wife's favorites with the kids.

I hope that I am not overstepping myself, because I love the software, and definitely enjoy playing a small part of this community, but I really enjoy the current functionality, and I am not certain that changing it is the best idea.

Thanks again.

**Comment by lachlan-00** [comment]
i'll flesh it out more here tomorrow. System playlists will alos revert to the user who runs the search at this time
